Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```diff
@@ -1,319 +1,122 @@
-}
-
-div.tabbedSubMenu span.tabStripTabSeparator
-{
-  display: none; /* magin-left must be set to 0 in the Non-IE browsers section for tabStripTab and tabStripTabSelected. */
+  min-height: calc(var(--remotion-themed-spacing) * 7);
 }
 
+div.tabbedMainMenu span.tabStripTab,
+div.tabbedMainMenu span.tabStripTabSelected,
 div.tabbedSubMenu span.tabStripTab,
 div.tabbedSubMenu span.tabStripTabSelected
 {
-  border-right: solid 1px #AAAAAA;
-
-  padding-top: 0.1em;
-  padding-left: 0.2em;
-  padding-right: 0.2em;
-
-  min-height: 1.4em;
-  min-height: calc(16px + 0.2em);
-}
-
-div.tabbedSubMenu div.tabStripTabsPane span.tabStripTabWrapper
-{
-  /* Compensates for 1px offset due to rounding errors when calculating the height in Internet Explorer. 
-    This might only be an issue if the TabbedMenu is displayed at the top of the page and manifests itself by jumping when moving the keyboard focus out of the TabbedMenu. */
-  padding-top: 0.1em;
-  padding-bottom: 0.1em;
+  padding: 0;
+  border: none;
+  border-radius: 0;
+  border-bottom: solid calc(var(--remotion-themed-spacing) * 0.75) transparent;
+  background-color: transparent;
+  align-items: center;
+  min-height: calc(var(--remotion-themed-spacing) * 7);
+  box-sizing: border-box;
 }
 
-div.tabbedSubMenu span.tabStripTab a,
-div.tabbedSubMenu span.tabStripTabSelected a
+div.tabbedMainMenu span.tabStripTab:not(.disabled):hover,
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled):hover,
+div.tabbedMainMenu span.tabStripTab:not(.disabled):focus-within,
+div.tabbedSubMenu span.tabStripTab:not(.disabled):hover,
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled):hover,
+div.tabbedSubMenu span.tabStripTab:not(.disabled):focus-within
 {
-  display: block;
-  position: relative;
-  min-height: 16px;
+  background-color: transparent;
+  border-bottom-color: var(--color-highlight);
 }
 
-div.tabbedSubMenu span.tabStripTab a span.anchorBody,
-div.tabbedSubMenu span.tabStripTabSelected a span.anchorBody
+div.tabbedMainMenu span.tabStripTab:not(.disabled):focus,
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled),
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled):focus,
+div.tabbedMainMenu span.tabStripTabSelected:not(.disabled):focus-within,
+div.tabbedSubMenu span.tabStripTab:not(.disabled):focus,
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled),
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled):focus,
+div.tabbedSubMenu span.tabStripTabSelected:not(.disabled):focus-within
 {
-  /* Compensates the margin-top: -0.1em for the img-tag */
-  padding-top: 0.1em;
+  border-bottom-color: var(--color-primary);
 }
 
-div.tabbedSubMenu span.tabStripTab a span.anchorBody > img,
-div.tabbedSubMenu span.tabStripTabSelected a span.anchorBody > img
+div.tabbedMainMenu div.tabStripTabsPane ul,
+div.tabbedSubMenu div.tabStripTabsPane ul
 {
-  position: relative;
-  top: -0.1em;
-  vertical-align: middle;
+  gap: calc(var(--remotion-themed-spacing) * 4);
```

Comment:
Imo the row and column gaps should have different values - the row gap should be smaller due to the `min-height` already being applied.

![image](https://user-images.githubusercontent.com/40357550/132318756-185bc648-a917-418e-a5a0-61a533d621e1.png)

Might need to check with @MichaelKetting